tests/test_extras_mm_fields.py

```diff
@@ -26,134 +26,136 @@ class MyBadObj:
+                required=False,
+            )
 
+        class LoadMyMmSchema(marshmallow.Schema):
             emisor_rut = RutField(
                 required=True,
-                load_from='RUT of Emisor',
+                data_key='RUT of Emisor',
             )
             other_field = marshmallow.fields.Integer(
                 required=False,
             )
 
         class MyMmSchemaStrict(marshmallow.Schema):
-            class Meta:
-                strict = True
 
             emisor_rut = RutField(
                 required=True,
-                load_from='RUT of Emisor',
+                data_key='RUT of Emisor',
             )
             other_field = marshmallow.fields.Integer(
                 required=False,
             )
 
         self.MyObj = MyObj
         self.MyBadObj = MyBadObj
-        self.MyMmSchema = MyMmSchema
+        self.LoadMyMmSchema = LoadMyMmSchema
+        self.DumpMyMmSchema = DumpMyMmSchema
         self.MyMmSchemaStrict = MyMmSchemaStrict
 
     def test_load_ok_valid(self) -> None:
-        schema = self.MyMmSchema()
+        schema = self.LoadMyMmSchema()
 
         data_valid_1 = {'RUT of Emisor': '1-1'}
         data_valid_2 = {'RUT of Emisor': Rut('1-1')}
         data_valid_3 = {'RUT of Emisor': ' 1.111.111-k \t '}
 
         result = schema.load(data_valid_1)
-        self.assertDictEqual(dict(result.data), {'emisor_rut': Rut('1-1')})
-        self.assertDictEqual(dict(result.errors), {})
+        self.assertDictEqual(dict(result), {'emisor_rut': Rut('1-1')})
 
         result = schema.load(data_valid_2)
-        self.assertDictEqual(dict(result.data), {'emisor_rut': Rut('1-1')})
-        self.assertDictEqual(dict(result.errors), {})
+        self.assertDictEqual(dict(result), {'emisor_rut': Rut('1-1')})
 
         result = schema.load(data_valid_3)
-        self.assertDictEqual(dict(result.data), {'emisor_rut': Rut('1111111-K')})
-        self.assertDictEqual(dict(result.errors), {})
+        self.assertDictEqual(dict(result), {'emisor_rut': Rut('1111111-K')})
 
     def test_dump_ok_valid(self) -> None:
-        schema = self.MyMmSchema()
+        schema = self.DumpMyMmSchema()
 
         obj_valid_1 = self.MyObj(emisor_rut=Rut('1-1'))
         obj_valid_2 = self.MyObj(emisor_rut=None)
 
-        data, errors = schema.dump(obj_valid_1)
+        data = schema.dump(obj_valid_1)
         self.assertDictEqual(data, {'emisor_rut': '1-1', 'other_field': None})
-        self.assertDictEqual(errors, {})
 
-        data, errors = schema.dump(obj_valid_2)
+        data = schema.dump(obj_valid_2)
         self.assertDictEqual(data, {'emisor_rut': None, 'other_field': None})
-        self.assertDictEqual(errors, {})
 
     def test_dump_ok_strange(self) -> None:
         # If the class of the object to be dumped has attributes that do not match at all the
         #   fields of the schema, there are no errors! Even if the schema has `strict = True` set.
```

Comment:
> Setting the `strict` option on `class Meta` has no effect on **`Schema`** behavior.

Source: https://marshmallow.readthedocs.io/en/stable/upgrading.html#schemas-are-always-strict

```suggestion
        #   fields of the schema, there are no errors!
```

- [ ] `RutFieldTest.test_dump_ok_strange()`
- [ ] `TipoDteFieldTest.test_dump_ok_strange()`
- [ ] `RcvTipoDoctoFieldTest.test_dump_ok_strange()`
- [ ] `RcvPeriodoTributarioFieldTest.test_dump_ok_strange()`